src/intro/hello-world.md

```diff
@@ -97,11 +91,11 @@ As an example, a Rust crate `hello` on Linux would be compiled to `libhello.so`:
 
 ```log
 $ cargo build
-   Compiling godot4-prebuilt v0.0.0 
-       (https://github.com/godot-rust/godot4-prebuilt?branch=4.1.1#fca6897d)
-   Compiling proc-macro2 v1.0.69
-   [...]
-   Compiling godot v0.1.0 (https://github.com/godot-rust/gdext?branch=master#66df8f47)
+   Compiling godot-cell v0.1.1
+   Compiling glam v0.27.0
+   Compiling godot-core v0.1.1
+   Compiling godot-ffi v0.1.1
+   Compiling godot v0.1.1
```

Comment:
I think this is a good example why we shouldn't include the log. It will already be outdated on next crates.io release again, and anytime we change dependencies, it becomes inaccurate. Furthermore, the output differs depending on the number of crates that have changed (it's not a full rebuild).

Could you remove the whole `cargo build` log?

The part about `ls -l` is also a bit strange, because `ls` doesn't typically show nested directories, nor does it put them in a frame. Maybe we should use a more conventional output here, following a `ls -l target/debug` command. But we can also just keep it OS-independent with a paragraph like

> After building, you should have the following files in the `target/debug` directory. Note that this applies to Linux, on Windows and macOS you will have different dynamic library names and file extensions.